code/datums/uplink_item.dm

```diff
@@ -542,6 +556,7 @@ var/list/uplink_items = list()
 	desc = "A speed loader that contains seven additional .357 Magnum rounds for the syndicate revolver. For when you really need a lot of things dead."
 	item = /obj/item/ammo_box/a357
 	cost = 2
+	excludefrom = list(/datum/game_mode/traitor/king_disk)
 
```

Comment:
Because I thought it'd be a good idea to do some balance changes at the time. Moving to a separate PR.